src/sankeyLink.js

```diff
@@ -0,0 +1,29 @@
+import {linkHorizontal, linkVertical} from "d3-shape";
+
+function horizontalSource(d) {
+  return [d.source.x1, d.y0];
+}
+
+function horizontalTarget(d) {
+  return [d.target.x0, d.y1];
+}
+
+export function sankeyLinkHorizontal() {
+  return linkHorizontal()
+      .source(horizontalSource)
```

Comment:
Very minor nit but indentation is 4 spaces here, should be 2 to match the rest of the code.

Reply:
Good catch, thanks :)

Reply:
Four spaces was correct here.

You are correct that two spaces is normally used for scope indentation (e.g., within a block), but four spaces is used for a statement that continues from the previous line, as with function chaining here. And in D3 style, if the result of the chained method changes the subject (e.g., *selection*.append, which returns a new selection, unlike *selection*.attr which returns the same *selection*), two spaces is used instead of four.